apps/ejabberd/src/mod_push_service_mongoosepush.erl

```diff
@@ -44,7 +44,8 @@
 %%--------------------------------------------------------------------
 
 -spec start(Host :: ejabberd:server(), Opts :: list()) -> any().
-start(Host, Opts) ->
+start(_ServerHost, Opts) ->
+    Host = ?MYNAME,
```

Comment:
I agree with @rslota, there is many installations where we need multiple XMPP host served by the same MongooseIM node.

The root cause is most probably in other place: https://github.com/esl/MongooseIM/pull/1222/files#diff-64730cfd8dcd5857c353e7ed3413f59dL108 The implementation now is bound to the main domain anyway, which is not a perfect solution. I somehow missed that while doing code review. The `Host` parameter which should be used is Publisher's host if I understand it correctly. @rslota, what's your opinion on that?

@astro could you give us more details what was the main issue for you here?